src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -351,6 +422,124 @@
           "faultCodeDetail"
         ]
       },
+      "TransactionInfo": {
+        "description": "Transaction data returned when querying for an existing transaction",
+        "type": "object",
+        "properties": {
+          "transactionId": {
+            "type": "string"
+          },
+          "payments": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/PaymentInfo"
+            },
+            "minItems": 1,
+            "maxItems": 1,
+            "example": [
+              {
+                "rptId": "77777777777302012387654312384",
+                "paymentToken": "paymentToken1",
+                "reason": "reason1",
+                "amount": 600,
+                "transferList": [
+                  {
+                    "paFiscalCode": "77777777777",
+                    "digitalStamp": false,
+                    "transferCategory": "transferCategory1",
+                    "transferAmount": 500
+                  },
+                  {
+                    "paFiscalCode": "11111111111",
+                    "digitalStamp": true,
+                    "transferCategory": "transferCategory2",
+                    "transferAmount": 100
+                  }
+                ]
+              }
+            ]
+          },
+          "status": {
+            "$ref": "#/components/schemas/TransactionStatus"
+          },
+          "feeTotal": {
+            "$ref": "#/components/schemas/AmountEuroCents"
+          },
+          "authToken": {
+            "type": "string"
+          },
+          "idCart": {
+            "description": "Cart identifier provided by creditor institution",
+            "type": "string",
+            "example": "idCartFromCreditorInstitution"
+          },
+          "clientId": {
+            "description": "transaction client id",
+            "type": "string",
+            "enum": [
+              "IO",
+              "CHECKOUT",
+              "CHECKOUT_CART",
+              "UNKNOWN"
+            ]
+          },
+          "nodeInfo":{
+            "type": "object",
+            "description": "Gateway infos",
+            "properties": {
+              "closePaymentResultError": {
+                "type": "object",
+                "description": "Error details for close payment result",
+                "properties": {
+                  "statusCode": {
```

Comment:
add description for statusCode and description fields

Reply:
done here 355365b0040f94471e6d70c9199b909bd64f02b0
replicating on other files too